blazor/diagram/swimlane/phase.md

```diff
@@ -0,0 +1,242 @@
+                    Height = 30
+                }
+            };
+        swimlane.Phases.Add(newPhase);
+    }
+
+    private void RemovePhase()
+    {
+        Swimlane swimlane = SwimlaneCollections[0];
+        if (swimlane.Phases.Count > 1)
+            swimlane.Phases.RemoveAt(swimlane.Phases.Count - 1);
+    }
+}
+```
+
+![Phase Add Remove](Swimlane-images/Phase_Add_Remove.gif)
+
+You can download a complete working sample from [GitHub]()
+
+## Create the Phase Header and Header customization
+
+* The `Header` property of Phase allows you to textually describe the phase and to customize the appearance of the description.
+* The size of Phase header can be controlled by using the `Width` and `Height` properties of header.
+* The appearance of Phase header can be customized by using the `Style` properties.
+
+The following code example explains how to define a Phase header and its customization.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+
+<SfDiagramComponent Height="600px" Swimlanes="@SwimlaneCollections" />
+
+@code
+{
+    //Define diagram's swimlane collection
+    DiagramObjectCollection<Swimlane> SwimlaneCollections = new DiagramObjectCollection<Swimlane>();
+
+    protected override void OnInitialized()
+    {
+        // A swimlane is created and stored in the swimlanes collection.
+        Swimlane swimlane = new Swimlane()
+            {
+                Header = new SwimlaneHeader()
+                {
+                    Annotation = new ShapeAnnotation()
+                    {
+                        Content = "SALES PROCESS FLOW CHART"
+                    },
+                    Height = 50,
+                },
+                OffsetX = 400,
+                OffsetY = 200,
+                Height = 150,
+                Width = 450,
+                Phases = new DiagramObjectCollection<Phase>()
+                {
+                    new Phase()
+                    {
+                        Width = 450,
+                        Header = new SwimlaneHeader()
+                        {
+                            Annotation = new ShapeAnnotation(){Content = "Phase 1", Style = new TextStyle(){ Color = "White", TextDecoration = TextDecoration.Underline, Italic = true, Bold = true} },
+                            Height = 30
+                        },
+                        Style = new TextStyle(){Fill = "Teal"},
+                    }
+                }
+            };
+        // Add swimlane
+        SwimlaneCollections.Add(swimlane);
+    }
+}
+```
+![Phase Header](Swimlane-images/Swimlane_Phase_Header.PNG).
+
+You can download a complete working sample from [GitHub]()
+
+## Header Selection and Resize
+
+ * You can select the individual phase header by clicking on the header twice. On the first click, you can select the respective phase. 
+
+ * You can able to resize the individual phase header. While resizing the phase, it will maintain 20px distances from Lane children.
```

Comment:
* You can resize the individual phase header. While resizing the phase, it will maintain 20px distances from Lane children.